File: app/src/main/java/com/org/jbossoutreach/Adapters/ContributorListAdapter.java
package com.org.jbossoutreach.Adapters;

import android.content.Context;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.bumptech.glide.Glide;
import com.org.jbossoutreach.Models.ContributionModel;
import com.org.jbossoutreach.R;

import java.util.ArrayList;

import androidx.annotation.NonNull;
import androidx.appcompat.widget.AppCompatImageView;
import androidx.appcompat.widget.AppCompatTextView;
import androidx.recyclerview.widget.RecyclerView;

public class ContributorListAdapter extends RecyclerView.Adapter<ContributorListAdapter.ViewHolder> {
    Context context;
    ArrayList<ContributionModel> list;

    public ContributorListAdapter(ArrayList<ContributionModel> list, Context context) {
        this.list = list;
        this.context = context;
    }


    @NonNull
    @Override
    public ContributorListAdapter.ViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
        View v = LayoutInflater.from(parent.getContext()).inflate(R.layout.contributorlistitem, parent, false);
        ContributorListAdapter.ViewHolder viewHolder = new ContributorListAdapter.ViewHolder(v);
        return viewHolder;
    }

    @Override
    public void onBindViewHolder(@NonNull ContributorListAdapter.ViewHolder holder, int position) {

        holder.Name.setText(list.get(position).getName());
        holder.Handle.setText(("@" + list.get(position).getName()));
        Glide.with(context)
                .load(list.get(position).getAvatar())
                .into(holder.Userimage);

    }

    @Override
    public int getItemCount() {
        return list.size();
    }

    public class ViewHolder extends RecyclerView.ViewHolder {

        public AppCompatTextView Name;
        public AppCompatTextView Handle;

        public AppCompatImageView Userimage;


        public ViewHolder(View itemView) {
            super(itemView);

            Name = (AppCompatTextView) itemView.findViewById(R.id.contributorname);
            Handle = (AppCompatTextView) itemView.findViewById(R.id.contributorhandle);
            Userimage = (AppCompatImageView) itemView.findViewById(R.id.contributorimage);


        }


    }
}

File: app/src/main/java/com/org/jbossoutreach/Adapters/ContributorListAdapter.kt
package com.org.jbossoutreach.Adapters

import android.content.Context
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.appcompat.widget.AppCompatImageView
import androidx.appcompat.widget.AppCompatTextView
import androidx.recyclerview.widget.RecyclerView
import com.bumptech.glide.Glide
import com.org.jbossoutreach.Models.ContributionModel
import com.org.jbossoutreach.R
import java.util.*

class ContributorListAdapter(var list: ArrayList<ContributionModel?>?, var context: Context) : RecyclerView.Adapter<ContributorListAdapter.ViewHolder>() {
    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
        val v = LayoutInflater.from(parent.context).inflate(R.layout.contributorlistitem, parent, false)
        return ViewHolder(v)
    }

    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
        holder.Name.text = list!![position].getName()
        holder.Handle.text = "@" + list!![position].getName()
        Glide.with(context)
                .load(list!![position].getAvatar())
                .into(holder.Userimage)
    }

    override fun getItemCount(): Int {
        return list!!.size
    }

    inner class ViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
        var Name: AppCompatTextView
        var Handle: AppCompatTextView
        var Userimage: AppCompatImageView

        init {
            Name = itemView.findViewById<View>(R.id.contributorname) as AppCompatTextView
            Handle = itemView.findViewById<View>(R.id.contributorhandle) as AppCompatTextView
            Userimage = itemView.findViewById<View>(R.id.contributorimage) as AppCompatImageView
        }
    }
}